code/src/App.js

```diff
@@ -1,9 +1,68 @@
 import React from 'react'
+import { LIST_URL } from 'utils/urls'
+import { BrowserRouter, Switch, Route } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import Header from 'components/Header'
+import MovieList from './components/MovieList'
+import MovieDetails from './components/MovieDetails'
+import LoadingSpinner from 'components/LoadingSpinner'
+import Footer from 'components/Footer'
+import ErrorMessage from 'components/ErrorMessage'
 
 export const App = () => {
+  const [movieTitle, setMovieTitle] = useState([])
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    fetchMovieTitle()
+  }, [])
+
+  const fetchMovieTitle = () => {
+    setLoading(true)
+    fetch(LIST_URL)
+      .then((res) => {
+        if (res.ok) {
+          return res.json()
+        } else {
+          throw new Error('something went wrong!')
+        }
+      })
+      .then((data) => setTimeout(() => setMovieTitle(data.results), 1000))
+      .finally(() => setTimeout(() => setLoading(false), 1000))
```

Comment:
Loader is amazing , as well as overall styling of the app . I love the styling for all the headings , it looks fancy ! A side note , when I was trying to press on the movie to get to the movie details page the very first time I had to wait for 4 seconds. The API itself is not that fast so I would suggest to decrease the loader timeout to 500ms, in summary it will appear as a faster user experience